tests/research/test_b2_03_impulse_morphology.py

```
@@ -0,0 +1,1026 @@
+                        "primary_positive": False,
+                        "material_relative_mae": False,
+                        "bootstrap_positive": False,
+                        "placebo_separation": False,
+                        "morphology_ordering": False,
+                    },
+                }
+            )
+    zero_cell = dict(cells[0])
+    zero_cell["per_cell_gates"] = {
+        name: False for name in lib.PER_CELL_GATE_NAMES
+    }
```

Ruff (0.16.3)

[warning] Unnecessary dict comprehension for iterable; use `dict.fromkeys` instead

Replace with `dict.fromkeys(iterable)`)

(C420)